tests/src/test/java/org/jboss/windup/tests/bootstrap/migrate/DisableTattletaleParameterTest.java

```diff
@@ -0,0 +1,53 @@
+package org.jboss.windup.tests.bootstrap.migrate;
+
+import org.jboss.windup.bootstrap.Bootstrap;
+import org.jboss.windup.rules.apps.tattletale.TattletaleRuleProvider;
```

Comment:
it seems this import is not used

Reply:
good catch, thanks.
Also [TattletaleEnablementByDefaultTest.java](https://github.com/windup/windup/pull/1330/files#diff-aedaa08448dcdd9e9b21f04ae02e2794) seems to have some